services/src/handlers/plots.rs

```diff
@@ -169,42 +167,51 @@ async fn get_plot_handler<C: Context>(
     };
 
     let query_ctx = ctx.query_context()?;
+    let task_manager = ctx.task_manager();
```

Comment:
I guess we shouldn't put it here but in all the pro handlers, right?

Reply:
30a6c58dfaebf7bfc8c5bb4501ed48684c9a64a8
7cd7fd0d2733776901d4ee8aebdcf8191cbc5fe1